src/views/Generator/TestOptions/LoadZones/LoadZones.tsx

```diff
@@ -48,6 +78,20 @@ export function LoadZones() {
     return () => subscription.unsubscribe()
   }, [watch, handleSubmit, onSubmit])
 
+  // evenly distribute load zones if distribution is set to "even"
+  useEffect(() => {
+    if (distribution !== 'even') return
+
+    const basePercent = Math.floor(100 / fields.length)
+    const remainder = 100 % fields.length
+
+    fields.forEach((_, index) => {
+      // ensure only integers are used
+      const percent = index < remainder ? basePercent + 1 : basePercent
+      setValue(`loadZones.${index}.percent`, percent)
+    })
```

Comment:
I'm following the same approach used in the cloud where only integers are being used and when "evenly" is selected, floating points are distributed to some items in the array, so not all of them will have the same percentage. For example:

![image](https://github.com/user-attachments/assets/65304f90-f9b5-457c-8f0f-b88b6772df19)


instead of al of them having `33.33%`. This makes it easier for users to match and manage the required sum of `100%`.